src/python_testing/test_metadata.yaml

```diff
@@ -97,6 +97,7 @@ slow_tests:
     - { name: TC_PS_2_3.py, duration: 30 seconds }
     - { name: TC_RR_1_1.py, duration: 25 seconds }
     - { name: TC_SWTCH.py, duration: 1 minute }
+    - { name: TC_TestAttrAvail.py, duration: 30 seconds }
```

Comment:
Does this test actually take that long?

Reply:
This takes a bit of time, maybe not entirely 30 seconds, but due to the commissioning for runs 1 and 2 it does take some time to complete.